pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -101,6 +104,21 @@ type CiliumNetworkPolicyNodeStatus struct {
 	Annotations map[string]string `json:"annotations,omitempty"`
 }
 
+type CiliumChildPolicyStatus struct {
```

Comment:
Can you just map it to `CiliumNetworkPolicyNodeStatus`?

Reply:
No, because the node status happens on Cilium and will be overwritten.